strapi/strapi
# [Content Manager] Add the new JSONInput from the DS to the CM

### What does it do?

- Replaces the current json input component with the one migrated to the design system
- Removes the old json input from the monorepo

### Why is it needed?

- To completely replace the old input

### How to test it?

- Open a content-type with a JSON field
- You should see the new JSONInput
- It should behave the same as the old input

<img width="652" alt="Screenshot 2023-01-27 at 11 48 59" src="https://user-images.githubusercontent.com/26598053/215068466-a86473fc-2007-4246-9610-53d29394d45c.png">
